src/app/tutorials/angular2-whatsapp-ionic-cli/index.ts

```diff
@@ -15,7 +15,7 @@ export const ANGULAR2_WHATSAPP_IONIC_CLI: TutorialDefinition = {
   },
   versions: {
     // Git revision => version definition object
-    'master': V2_0_0,
+    'master': V3_0_0,
```

Comment:
???
AFAIK master contains the latest version of the tutorial and contains all my commits: https://github.com/Urigo/Ionic2CLI-Meteor-WhatsApp/commits/master
master-history instead contains a single commit for each version of the tutorial.
Also, Tortilla creates a tag in the master branch for each version and each major step of the tutorial.